src/index.ts

```diff
@@ -1,15 +1,22 @@
 import express, { Application, Request, Response } from 'express';
+import userRoutes from './routes/userRoutes';
+import todoRoutes from './routes/todoRoutes';
+import { errorHandler } from './middlewares/errorHandler';
+import connectDB from './database';
 
 const app: Application = express();
-const port: number = 3000;
 
-// A simple get request
-app.get('/', (request: Request, response: Response)=>{
-    response.status(200).send("Hello World");
+// Connect to MongoDB
+connectDB();
 
-})
+app.use(express.json());
```

Comment:
move this statement higher. Do read why this should be higher in your middleware order